content/en/docs/reference/access-authn-authz/certificate-signing-requests.md

```diff
@@ -156,24 +170,30 @@ To allow signing a CertificateSigningRequest:
 
 {{< codenew file="access/certificate-signing-request/clusterrole-sign.yaml" >}}
 
-## Normal User
+## Normal user
 
-A few steps are required in order to get normal user to be able to authenticate and invoke an API. First, this user must have certificate issued by the Kubernetes Cluster, and then present that Certificate to the API call as the Certificate Header or through the kubectl.
+A few steps are required in order to get normal user to be able to
```

Comment:
Is it "Normal user" or something like: "Normal [or common] authentication workflow"?
nit: Under normal conditions, there are a few steps required for a user to authenticate and invoke an API.